src/platform/web/ui/session/room/timeline/GapView.js

```diff
@@ -29,10 +29,25 @@ export class GapView extends TemplateView {
             isLoading: vm => vm.isLoading,
             isAtTop: vm => vm.isAtTop,
         };
-        return t.li({className}, [
-            spinner(t),
-            t.div(vm => vm.isLoading ? vm.i18n`Loading more messages …` : vm.i18n`Not loading!`),
-            t.if(vm => vm.error, t => t.strong(vm => vm.error))
+        return t.li({ className }, [
```

Comment:
I think this could probably be simplified to have a binding on showing the spinner (e.g. a hidden class binding) and have a single description property on the vm which returns the message to be display (either or or normal operation) and bind to that here.